main.go

```diff
@@ -0,0 +1,175 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	// to ensure that exec-entrypoint and run can make use of them.
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/cache"
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	metricsserver "sigs.k8s.io/controller-runtime/pkg/metrics/server"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+
+	k8sv1 "github.com/cloudogu/k8s-blueprint-operator/pkg/api/v1"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/config"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/controller"
+	//+kubebuilder:scaffold:imports
+)
+
+var (
+	// Version of the application
+	Version = "0.0.0"
+)
+
+var (
+	scheme   = runtime.NewScheme()
+	setupLog = ctrl.Log.WithName("setup")
+
+	// These variables are here to avoid errors during leader election.
+	leaseDuration = time.Second * 60
+	renewDeadline = time.Second * 40
+)
+
+func init() {
+	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
+
+	utilruntime.Must(k8sv1.AddToScheme(scheme))
+	//+kubebuilder:scaffold:scheme
+}
+
+func main() {
+	ctx, cancel := NoInheritCancel(nil)
```

Comment:
🚫 **[checkstyle]** <staticcheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>SA1012: do not pass a nil Context, even if a function permits it; pass context.TODO if you are unsure about which Context to use